### ISSUE TITLE
Anyway to reduce binary size compiled with bitcode?

### ISSUE THREAD

**Issue by myonlylonely** [kakashidinho/metalangle#80]
With bitcode compiled, the binary size is about 1 GB. Is there any way to reduce that size? I need to compile my App with bicode, so I have to recompile MetalANGLE framework with bitcode, but the binary size is so big.

**Comment by kakashidinho** [comment]
Hi,
what did you do to change MetalANGLE to be compiled with bitcode? I believe the provided Xcode project already leaves the bitcode option as is (with default value = enabled). i.e. it is already compiled with bitcode.

Furthermore, are you using static library version of MetalANGLE?

**Comment by kakashidinho** [comment]
That's new. I wasn't aware.
Btw, the bitcode binary size is not the final size. When you upload your binary to appstore, it should be compiled one more into native binary and the size should be reduced.

**Comment by myonlylonely** [comment]
> That's new. I wasn't aware. Btw, the bitcode binary size is not the final size. When you upload your binary to appstore, it should be compiled one more into native binary and the size should be reduced.

Yes, I know that doesn't affect the final size of the App. But it makes the upload process to Apple Store Connect very slow.

**Comment by danoli3** [comment]
Just target arm64 with bitcode
